File: C/1000-1999/1010.c
#include <stdio.h>

int main (void){
  int cod, uni, cod2, uni2;
  float valor, valor2, res;

  scanf("%d %d %f", &cod, &uni, &valor);
  scanf("%d %d %f", &cod2, &uni2, &valor2);

  res = uni*valor + uni2*valor2;

  printf("VALOR A PAGAR: R$ %.2f\n", res);


  return 0;
}

File: Python/1000-1999/1010.py
# -*- coding: utf-8 -*-

def main():
    l1 = input().split(" ")
    l2 = input().split(" ")
    c1, n1, v1 = l1
    c2, n2, v2 = l2
    r = (int(n1) * float(v1) + int(n2) * float(v2))
    print("VALOR A PAGAR: R$ %.2f" %r)
main()